webapp/commonComponents/hooks/useAsyncPutRequest.js

```diff
@@ -3,4 +3,4 @@ import axios from 'axios'
 
 import useAsync from './useAsync'
 
-export default (url, config = {}) => useAsync(axios.put, [url, config])
\ No newline at end of file
+export default (url, data = {}, config = {}) => useAsync(axios.put, [url, data, config])
```

Comment:
you are changing the order of the params. does this affect where it was used?

Reply:
It's only used in one place